src/site/content/en/vitals/index.md

```diff
@@ -1,13 +1,12 @@
 ---
-layout: page
+layout: post
 title: Web Vitals
-subhead: Essential metrics for a healthy site
 description: Essential metrics for a healthy site
+hero: web-vitals-hero.png
 authors:
   - philipwalton
```

Comment:
@philipwalton we should remove your author attribution here? It makes the page seem more authoritative.

Reply:
I go back and forth on this. The benefit of attribution is readers know who to reach out to for questions, but I agree it does make it look a bit more like a blog post (though I'm not sure how much of that is the author image vs. the overall layout).